pkg/apis/v1alpha5/provisioner.go

```diff
@@ -97,12 +99,19 @@ type Consolidation struct {
 // +kubebuilder:object:generate=false
 type Provider = runtime.RawExtension
 
+func ProviderAnnotation(p *Provider) map[string]string {
```

Comment:
thoughts on baking this into the json serialization so no one needs to call this?

Reply:
hmm -- maybe better as you've done it.